docs/blog/posts/2023-09-18-rclone.md

```diff
@@ -1,25 +1,18 @@
 ---
 date:
   created: 2023-09-18
-pin: true
+categories:
+    - File Transfer
+    - rclone
+    - Google Drive
 ---
+# Transferring files from the Yens to Google Drive using `rclone`
```

Comment:
Can we shorten the title to fit into one line? Maybe something like "rclone files from Yens to Google drive"